src/model.py

```diff
@@ -28,16 +28,19 @@
 import json
 import os
 import threading
-from typing import AsyncGenerator
+from typing import AsyncGenerator, Dict, List
 
 import numpy as np
 import triton_python_backend_utils as pb_utils
-from vllm import SamplingParams
+from vllm.sampling_params import SamplingParams     # MODIFY: bug fix, it won't bother
```

Comment:
could you please clarify this bug?

Reply:
I'm using `pip install -e .` to install vLLM from their source code (to test multi-lora version), in this case, using `from vllm import SamplingParams` will throw `ImportError`: 

```
ImportError: cannot import name 'SamplingParams' from 'vllm' (unknown location)
```

So I just tried to import the `SamplingParams` from the source code directly, to avoid this problem.

---

I think this modification will not affect both common users and developers, and solve the problem caused by `ImportError`.

Reply:
Sounds good. I'll keep this conversation unresolved for history